node/src/components/chainspec_loader.rs

```diff
@@ -133,12 +133,6 @@ impl Display for NextUpgrade {
     }
 }
 
-/// Basic information about the current run of the node software.
-#[derive(Clone, Debug)]
-pub(crate) struct CurrentRunInfo {
-    pub(crate) last_emergency_restart: Option<EraId>,
-}
```

Comment:
I'm a bit surprised that this is the only information we're reporting for `CurrentRunInfo`... but also, would it be brought back as part of @afck 's work?

Reply:
I don't think so, no.

Reply:
I think it might have carried more information in the past... I don't know if @afck will be bringing it back, it might no longer be necessary (this was really just a way to read some stuff from the chainspec via the `EffectBuilder`, and the necessary information will now be read from the global state instead).